checks/normandy/jexl_error_rate.py

```diff
@@ -21,13 +24,29 @@
     min_timestamp = min(r["min_timestamp"] for r in rows)
     max_timestamp = max(r["max_timestamp"] for r in rows)
 
-    total = sum(row["total"] for row in rows)
-    classify_errors = sum(
-        row["total"] for row in rows if row["status"] == "content_error"
-    )
-    error_rate = classify_errors * 100.0 / total
+    # The Redash query returns statuses by periods (eg. 10min).
+    # First, agregate totals by period.
+    periods: Dict[Tuple[str, str], Dict] = {}
```

Comment:
I think this would work nicer as a defaultdict with a `Counter` as the default value

```python
from collections import Counter
periods = defaultdict(Counter)
```

I'm not sure how to do the typing on that though

Reply:
Awesome :) TIL about `Counter`!